### DISCUSSION TITLE
Copy link for linked branch on issue page

### DISCUSSION THREAD

**Discussion by da-mask** [community/community#37829] [Projects and Issues]
When on an issue page, I find if very difficult to find an easy way to copy the linked branch so I can check it out in my terminal.  I think it would be very convenient to have a copy link in the `Development` section of the issue page, does anyone else find this? Maybe I'm missing an easy way to access the branch for copying?

**Comment by ossanna16** [comment]
Hi there @da-mask and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
